crates/core/tedge_actors/src/messages.rs

```diff
@@ -0,0 +1,7 @@
+use std::fmt::Debug;
+
+/// A message exchanged between two actors
+pub trait Message: Debug + Send + Sync + 'static {}
```

Comment:
Wondering if this couldn't just be a `type` alias which we use in the generic type constraints of `Input` and `Output` in the `Actor` trait.

Reply:
A type alias is concrete and cannot be used as a type constraint.